README.md

```diff
@@ -60,11 +107,13 @@ export PUBSUB_EMULATOR_HOST=::1:8410
 ```bash
 cat > .env << EOS
 RABBIT_AMQP=amqp://guest:guest@localhost:6672
-GCP_PROJECT_ID=project
+SUBSCRIPTION_PROJECT_ID=project
+RECEIPT_TOPIC_PROJECT_ID=project
 PUBSUB_EMULATOR_HOST=localhost:8410
```

Comment:
maybe mention that this port is the one outputted from the pubsub startup - as it looks like its always 8410

Reply:
Done.